scenarios/iam_privesc_by_key_rotation/cheat_sheet.md

```diff
@@ -1,34 +1,31 @@
 # IAM Privecs by Key Rotation Cheat Sheet
```

Comment:
Could you replace `Kerrigan` with `manager`

I realized when I copied over the default output name from another scenario. Fix in #282

Reply:
Okay. I'll change to manager profile.